apps/frontend/src/pages/auth/libs/components/forgot-password-form/forgot-password-form.tsx

```diff
@@ -37,7 +39,11 @@ const ForgotPasswordForm: React.FC<Properties> = ({ onSubmit }: Properties) => {
 					type="email"
 				/>
 
-				<Button label="SEND LINK" type="submit" />
+				<Button
+					isDisabled={email === ""}
```

Comment:
The submission will not happen even if the user clicks on button, because validation schema will not allow it. In your current approach `getValues` is not reactive, so I believe that won't work

Reply:
What else can I use for this button to be empty when the input field is empty? Or should I just leave the button clickable and rely on the validation?

Reply:
I think relying on validation is enough. The main thing is that the request will not be sent to Backend